File: BaekJoon/string/2675.py
import sys

count = int(sys.stdin.readline())

for _ in range(count):
    result = ""
    num, string = sys.stdin.readline().split()
    num = int(num)
    if (num == 0):
        continue
    for i in range(len(string)):
        result += (string[i]*num)
    print(result)

File: BaekJoonSwift/BaekJoonSwift/String/2675.swift
import Foundation

let n = Int(readLine()!)!

for _ in 0..<n {
    let iValues = readLine()!.split(separator: " ")
    let count = Int(iValues[0])!
    let string = Array(iValues[1])
    
    for char in string {
        for _ in 0..<count {
            print(char, terminator: "")
        }
    }
    print()
    
}
